packages/stripe_ios/ios/Classes/StripeSdk.swift

```diff
@@ -169,7 +169,7 @@ public class StripeSdk: RCTEventEmitter, STPApplePayContextDelegate, STPBankSele
                 self.paymentSheet?.present(from: UIApplication.shared.delegate?.window??.rootViewController ?? UIViewController()) { paymentResult in
                     switch paymentResult {
                     case .completed:
-                        resolve([])
+                        resolve([String: String]())
```

Comment:
can you revert this to not divert from react native sources?

Reply:
the problem is that React native is not consistent. On Android they return an empty map and on iOS an empty list. I can do the conversion in dart layer without issues but it will be a bit ugly